Library/Formula/bash.rb

```diff
@@ -1,19 +1,20 @@
 class Bash < Formula
   desc "Bash (Bourne-again SHell) is a UNIX command interpreter"
   homepage "https://www.gnu.org/software/bash/"
+  head "http://git.savannah.gnu.org/r/bash.git"
 
   stable do
     url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz"
     mirror "https://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz"
     sha256 "afc687a28e0e24dc21b988fa159ff9dbcf6b7caa92ade8645cc6d5605cd024d4"
-    version "4.3.33"
+    version "4.3.39"
 
     # Vendor the patches. The mirrors are unreliable for getting the patches,
     # and the more patches there are, the more unreliable they get. Upstream
     # patches can be found in: http://git.savannah.gnu.org/cgit/bash.git
     patch do
-      url "https://gist.githubusercontent.com/jacknagel/c1cf23775c774e2b4b6d/raw/838118bf0e07dcde8a552cb8556600b7e453d4ac/bash-4.3.33.diff"
-      sha1 "71fc36bced0d15a2c221cfcbff02ea412d4bf5fa"
+      url "https://gist.githubusercontent.com/anonymous/4282245753d5348e438b/raw/8402dc95200af1edcad1528d61d07d7003301c4b/bash-4.3.39.diff"
```

Comment:
Savannah's git can be a little up-and-down, which may be why we've avoided doing so so far? If it's less grumpy these days, turning it into a git clone could be ideal.